containers/scripts/init-db.sh

```diff
@@ -40,22 +40,9 @@ main() {
 
 minimal() {
     set -ex
-
-    # weak password used for testing purposes only
-    PASSWD=xxxxxx
-
     # grant CREATEDB priviledges needed for tests
     podman exec db psql -h localhost -U openscanhub -c 'ALTER USER openscanhub CREATEDB;'
-
-    # create OpenScanHub users
```

Comment:
Why is this comment being removed?

Reply:
I don't see it really helpful at the very begining. anyway I can add it back.